### PR TITLE
chore: refactor static content

### PR DESCRIPTION
**Pull request by dlnanh85**
- Move all static content into `src/data`
- Update imports accordingly

### CODE REVIEW

File: src/data/myUrl.ts
Comment (NooBat) [inline src/data/myUrl.ts:1]: Can you explain to me why this file is named myUrl instead of src/data/urlList.ts?

File: src/types/project-type.ts
Comment (NooBat) [inline src/types/project-type.ts:1]: Why is this file's name following kebab-case not camelCase?

File: src/types/tool-type.ts
Comment (NooBat) [inline src/types/tool-type.ts:1]: Why is this file's name following kebab-case not camelCase?

**Review by NooBat — COMMENTED** [review]
Please provide me with a brief explain behind your reasoning 🤔

File: src/data/myUrl.ts
Comment (dlnanh85) [reply src/data/myUrl.ts:1]: As for the other files, I strictly follow the naming convention if there's only one export for the file. This file, however, exports multiple objects with different names, so I need a more general name, and this file is mainly used for MyURL page, therefore, `myUrl`. If you think it still should be `urlList.ts`, please let me know. Thank you.

**Review by NooBat — APPROVED** [review]
LGTM!

**Pull request merged**